packages/web-components/src/components/va-button/va-button.css

```diff
@@ -0,0 +1,61 @@
+@import '../../mixins/buttons.css';
+
+:host {
+  display: inline-block;
+}
+
+.va-button {
+  align-items: center;
+  background-color: var(--color-primary);
+  border-radius: 5px;
+  color: var(--color-white);
+  display: flex;
+  font-size: 1.6rem;
+  margin: 0.5em 0.5em 0.5em 0;
+}
+
+.va-button:active,
+.va-button:hover {
+  background-color: var(--color-primary-darker);
+}
+
+.va-button.secondary {
+  background-color: var(--color-white);
```

Comment:
Addresses https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/196 ✅